QUEST.tsv

```diff
@@ -628,8 +628,8 @@ QUEST_20150317_000627	When the great swordsman, Ruklys, completed the fortress,
 QUEST_20150317_000628	Their power grew to that of a city-state over time,{nl}and it is suspected that Ruklys' ambition grew as well.
 QUEST_20150317_000629	Ruklys' influence grew so much that he eventually revolted against King Kadumel.
 QUEST_20150317_000630	It was a huge rebellion. The entire city took part. {nl}That's why some record it as a civil war.
-QUEST_20150317_000631	The rebellion initially favored Ruklys, but tides turned {nl}when the great Premier Eminent and marksman Lydia Schaffen joined the war.
-QUEST_20150317_000632	It all ended two years after, when Ruklys died at the hands of Lydia Schaffen.
+QUEST_20150317_000631	The rebellion initially favored Ruklys, but, the tides turned {nl}when the great Premier Eminent and marksman Lydia Schaffen joined the war.
```

Comment:
On the top of this page, right after the green Open label, see how it tells you:

stvkno  wants to merge 1 commit into Treeofsavior:master from stvkno:patch-1

On Github, you make all your changes in a forked repository, then you can ask others to 'pull' your changes to their repository. 

In your forked repository, you can make different 'branches' that pertains to different 'versions' of your changes. On this PR, you have made changes to a branch called "patch-1", and you want to merge those changes in "patch-1" to "master" branch of TreeofSavior.

So, tldr; you go to your forked repository https://github.com/stvkno/EnglishTranslation
on the page should be a Branch: master button near the top left. That button will expand into other branches that you have. And you can select those branches (like patch-1, patch-2, patch-3 in your case) of your PRs and edit from there.

If you want to really get into Github though, you should consider learning git commands and using a git client (git GUI or git bash) https://git-scm.com/downloads

Hope this helps!